File: c/if_else/ProfitLoss.c
#include<stdio.h>
int main(){
    int x,y,z;
    printf("entor cost price:");
    scanf("%d",&x);
    printf("enter shelling price:");
    scanf("%d",&y);
    if(y>x){
        printf ("you got profit and");

    }
    else{
        printf("you got loss and");
    }
    z=y-x;
    printf("\n your profit money is : %d ",z);
    return 0;
}

File: python/If_else/Profit_Loss.py
cp=int(input("enter cost price: "))
sp=int(input("enter shelling price:"))
if cp<sp:
    print("you got profit")
elif cp==sp:
    print("No Profit no Loss")
else:
    print("you got loos") 
profit=sp-cp
print("your profit or loss money is:",profit)
  